api/ai/new/interfaces/algorithm_options.py

```diff
@@ -72,3 +73,11 @@ def validate(self):
 class PathGaspAlgorithmOptions(AlgorithmOptions):
     def validate(self):
         super().validate()
+
+
+@dataclass
+class MultipleRoundRobinAlgorithmOptions(AlgorithmOptions):
+    projects: List[Project]
+
+    def validate(self):
+        super().validate()
```

Comment:
you can do this as another ticket, but are there no constraints on the list of projects? like if it's empty does the algorithm run fine? if not you need to add those validations here